pydra/tasks/mriqc/workflows/anatomical/base.py

```diff
@@ -982,33 +956,32 @@ def airmsk_wf(name="AirMaskWorkflow"):
         ),
         name="inputnode",
     )
-    outputnode = pe.Node(
-        niu.IdentityInterface(fields=["hat_mask", "air_mask", "art_mask", "rot_mask"]),
+    outputnode = pydra.Node(
+        interface=specs.IdentityInterface(fields=["hat_mask", "air_mask", "art_mask", "rot_mask"]),
         name="outputnode",
     )
 
-    rotmsk = pe.Node(RotationMask(), name="RotationMask")
-    qi1 = pe.Node(ArtifactMask(), name="ArtifactMask")
+    rotmsk = ShellCommandTask(name="RotationMask", executable="RotationMask")
```

Comment:
This line should be `workflow.add(RotationMask(name="rotmsk"))` NB: `rotmsk` is arbitrary but using the same name as the variable that was assigned to in the Nipype code will make the connections easier to track